cdap-master/src/test/java/co/cask/cdap/master/startup/ConfigurationCheckTest.java

```diff
@@ -62,4 +62,9 @@ public void invalidAuditTopicTest() {
   public void invalidLogPartition() {
     runConfigurationCheck(Constants.Logging.LOG_PUBLISH_PARTITION_KEY, "application1");
   }
+
+  @Test (expected = RuntimeException.class)
+  public void invalidHBaseDDLExecutorConfTest() {
```

Comment:
You need a test for the valid config as well to make sure it won't give false failure.